app/assets/javascripts/admin/addon/components/themes-grid-card.gjs

```diff
@@ -83,6 +83,8 @@ export default class ThemeCard extends Component {
       },
       duration: "short",
     });
+
+    window.location.reload();
```

Comment:
Btw, would we be able to do a `this.router.refresh();` here? I believe it might be a faster refresh.

Reply:
Good idea! I just checked, and sadly, it is not working. Themeid is calculated and added to <head> block. With ember reloa,d that value is not updated:

https://github.com/discourse/discourse/blob/main/app/views/layouts/application.html.erb#L7